src/lib/nvme_pt_ioctl.c

```diff
@@ -161,6 +161,10 @@ static uint8_t opal_method[][OPAL_UID_LENGTH] = {
 		{ 0x00, 0x00, 0x00, 0x06, 0x00, 0x00, 0x08, 0x03 },
 };
 
+static const uint8_t opal_mehod_params[][OPAL_UID_LENGTH] = {
+	[OPAL_REVERTSP_KEEP_GL_RN_KEY] = {0x83, 0x06, 0x00, 0x00},
```

Comment:
KeepGlobalRangeKey parameter is 0x00600000 , 0x83 byte should by using opal parser API (see append_short_atom* functions family)

Reply:
done